larray_editor/arraywidget.py

```diff
@@ -536,11 +648,81 @@ class ArrayEditorWidget(QWidget):
+        self.view_axes = LabelsView(parent=self, model=self.model_axes, position=(LEFT, TOP))
+
+        self.model_xlabels = LabelsArrayModel(parent=self, readonly=readonly)
+        self.view_xlabels = LabelsView(parent=self, model=self.model_xlabels, position=(RIGHT, TOP))
+
+        self.model_ylabels = LabelsArrayModel(parent=self, readonly=readonly)
+        self.view_ylabels = LabelsView(parent=self, model=self.model_ylabels, position=(LEFT, BOTTOM))
+
+        self.model_data = DataArrayModel(parent=self, readonly=readonly, bg_value=bg_value, bg_gradient=bg_gradient,
+                                         minvalue=minvalue, maxvalue=maxvalue)
+        self.view_data = DataView(parent=self, model=self.model_data, dtype=data.dtype, shape=data.shape)
+
+        self.mapping = MappingLArrayData(axes_model=self.model_axes, xlabels_model=self.model_xlabels,
+                                         ylabels_model=self.model_ylabels, data_model=self.model_data, data=data)
+
+        # Synchronize resizing with labels views
+        # self.view_xlabels.horizontalHeader().sectionResized.connect(self.view_ylabels.updateSectionWidth)
+        self.view_xlabels.horizontalHeader().sectionResized.connect(self.view_data.updateSectionWidth)
+        self.view_ylabels.verticalHeader().sectionResized.connect(self.view_data.updateSectionHeight)
+
+        # Synchronize scrolling
+        # data <--> xlabels
+        self.view_data.horizontalScrollBar().valueChanged.connect(self.view_xlabels.horizontalScrollBar().setValue)
+        self.view_xlabels.horizontalScrollBar().valueChanged.connect(self.view_data.horizontalScrollBar().setValue)
+        # dqtq <--> ylabels
+        self.view_data.verticalScrollBar().valueChanged.connect(self.view_ylabels.verticalScrollBar().setValue)
+        self.view_ylabels.verticalScrollBar().valueChanged.connect(self.view_data.verticalScrollBar().setValue)
+
+        # Synchronize selecting columns via hor. header of xlabels view
+        self.view_xlabels.horizontalHeader().sectionPressed.connect(self.view_data.selectColumn)
+        # Synchronize selecting rows via vert. header of ylabels view
+        self.view_ylabels.verticalHeader().sectionPressed.connect(self.view_data.selectRow)
+
+        # set style sheet
+        self.view_axes.setStyleSheet("""
+            border-top: 1px solid grey;
+            border-bottom: none;
+            border-left: 1px solid grey;
+            border-right: none;
+            """)
+        self.view_xlabels.setStyleSheet("""
+            border-top: 1px solid grey;
+            border-bottom: none;
+            border-left: none;
+            border-right: 1px solid grey;
+            """)
+        self.view_ylabels.setStyleSheet("""
+            border-top: none;
+            border-bottom: 1px solid grey;
+            border-left: 1px solid grey;
+            border-right: none;
+            """)
+        self.view_data.setStyleSheet("""
```

Comment:
changing the code to:
```python
    self.view_data.setStyleSheet("""QTableView {
            border-top: none;
            border-bottom: 1px solid grey;
            border-left: none;
            border-right: 1px solid grey;
    }""")
```
seems to do the trick (it does not apply the style to all child widgets)